src/python/era_5g_netapp_client/tests/test_client_gstreamer.py

```diff
@@ -15,50 +17,47 @@ def get_results(results: str):
     print(results)
     pass
 
+
 def main():
     """
     Creates the client class and starts the data transfer
     """
 
-    # IP address or hostname of the server
-    server_ip = "IP_OR_HOSTNAME"
-    user = "GUID"
-    password = "passwd"
-
+    # ip address or hostname of the server (middleware)
+    server_ip = "localhost"
+    user = "2c63955e-27cf-4b9a-980b-b70a54281d15"
+    password = "test"
     task_id = "7d93728a-4a4c-4dae-8245-16b86f85b246"
-    
-    # if middleware is omitted, the netapp address and port has to be specified manually
-    #netapp_uri = "127.0.0.1"
-    #netapp_port = "5896"  
+    # to avoid exception in "except"
+    client = None
 
     try:
-        # to avoid exception in "except"
-        client = None
         # creates the NetApp client with gstreamer extension
-        client = NetAppClientGstreamer(server_ip, user, password, task_id, True, get_results, True, True)
+        client = NetAppClientGstreamer(server_ip, user, password, task_id, "True", get_results, True, True)
         # register the client with the NetApp
         client.register()
-        # creates a data sender which will pass images to the NetApp either from webcamera ...
-                
-        data_src = f"v4l2src device=/dev/video0 ! video/x-raw, format=YUY2, width=640, height=480, " + \
-                    "pixel-aspect-ratio=1/1 ! videoconvert ! appsink"
-        sender = DataSenderGStreamerFromSource(client.netapp_host, client.gstreamer_port, data_src, 15, 640, 480, False)
-        
-        # ... or from file
-        
-        #sender = DataSenderGStreamerFromFile(client.netapp_host, client.gstreamer_port, 15, "/path/to/the/file.ext", 640, 480)
+        if FROM_SOURCE:
+            # creates a data sender which will pass images to the NetApp either from webcam ...
+            data_src = f"v4l2src device=/dev/video0 ! video/x-raw, format=YUY2, width=640, height=480, " + \
+                       "pixel-aspect-ratio=1/1 ! videoconvert ! appsink"
+            DataSenderGStreamerFromSource(client.netapp_host, client.gstreamer_port, data_src, 15, 640, 480,
+                                          False)
+        else:
+            # or from file
+            DataSenderGStreamerFromFile(client.netapp_host, client.gstreamer_port, 15,
+                                        "../../../../assets/2017_01_02_001021_s.mp4", 640, 480)
```

Comment:
možná spíš nastavit přes nějakou env proměnnou? Takto to bude (pokud se nepletu) závislé na tom, odkud ten skript pustím. Navíc celý era_5g_netapp_client se výhledově přesune do vlastního repozitáře, kdežto example video by mělo zůstat u referenční netapp...

Reply:
souhlas, env proměnná mi taky zní líp

Reply:
to pak ale musí být ten soubor součástí package ne? tady je mimo ve složce /assets